source/jormungandr/jormungandr/equipments/equipment_provider_manager.py

```diff
@@ -75,9 +75,6 @@ def _init_class(self, cls, arguments):
         :return: instance of provider
         """
         try:
-            if '.' not in cls:
-                self.logger.warning('impossible to build, wrongly formated class: {}'.format(cls))
```

Comment:
I don't get what's affected by that change, even though it looks harmless.

Reply:
It is most probable that there will be an exception and hence will write another log four lines after.
I simply deleted duplicate log writing.

Reply:
I think the change is that if we use a "root" class it won't raise anymore, but before it did... I guess it's not important.